src/spokanetech/settings.py

```diff
@@ -180,3 +182,15 @@
 PROJECT_NAME = "Spokane Tech"
 PROJECT_DESCRIPTION = """Community resource for all things tech in the Spokane and CDA area"""
 PROJECT_VERSION = "0.0.1"
+
+
+# Celery
+CELERY_BROKER_URL = os.environ["CELERY_BROKER_URL"]
+CELERY_RESULT_BACKEND = os.environ.get("CELERY_RESULT_BACKEND", "django-db")
```

Comment:
any particular reason to use django-db over redis here?

Reply:
[django-celery-results](https://pypi.org/project/django-celery-results/) lets you use the Django admin UI to view results which is convenient at times.